Hogajama/hogajama-angular-frontend/src/main/ionic/src/environments/environment.prod.ts

```diff
@@ -1,5 +1,5 @@
 const keycloakConfig = {
-  url: 'https://keycloak-hogarama.apps.play.gepaplexx.com/auth/',
+  url: 'https://keycloak-hogarama.apps.steppe.gepaplexx.com/auth/',
```

Comment:
unabhängig von dem PR, es schmerzt immer noch, dass die Keycloak Configuration hardcoded ist =/

Reply:
Du stocherst in einer tiefen offenen Wunde 😢